tests/build.gradle

```diff
@@ -57,6 +57,8 @@ dependencies {
     compile project(':common:scala')
     compile project(':core:controller')
     compile project(':core:invoker')
+
+    testCompile 'com.typesafe.akka:akka-http-testkit_2.11:10.0.9'
```

Comment:
Above deps are under "compile" for a reason: gradle has its problems sharing testDeps throughout projects, having them under compile makes them available to projects depending on tests.